test/katello_test_helper.rb

```diff
@@ -88,7 +88,7 @@ module FixtureTestCase
 
     Setting::Content.load_defaults
 
-    @@admin = ::User.find(FIXTURES['users']['admin']['id'])
+    @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
```

Comment:
so i think hound is throwing up an error here, we don't enable the class var cop, and now rubocop is failing with: 
`W: Unnecessary disabling of Style/ClassVars.`

in jenkins :( 

It seems like hound doesn't honor .rubocop.yaml files in subdirectories:  https://github.com/Katello/katello/blob/master/test/.rubocop.yml
hound--